src/components/DateTimePicker/index.js

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable react/prop-types */
+import React, { useRef, useState, useImperativeHandle } from 'react';
+import PropTypes from 'prop-types';
+import withReduxForm from '../../libs/hocs/withReduxForm';
```

Comment:
make a custom hook (useReduxForm) which will be the much the same as withReduxForm
place it inside libs/customHooks/useReduxForm./index.js